pkg/process/monitor/process_monitor.go

```diff
@@ -35,6 +36,43 @@ var (
 	}
 )
 
+// processMonitorTelemertry
+type processMonitorTelemertry struct {
+	// process monitor will process :
+	//  o events refer to netlink events received (not only exec and exit)
+	//  o exec process netlink events
+	//  o exit process netlink events
+	//  o restart the netlink connection
+	//
+	//  o events_channel_size is the queue length of netlink messages channel
+	//  o reinit_failed is the number of failed re-initialisation after a netlink restart due to an error
+	//  o process_scan_failed would be > 0 if initial process scan failed
+	//  o callback_called numbers of callback called
+	events  *telemetry.Counter
+	exec    *telemetry.Counter
+	exit    *telemetry.Counter
+	restart *telemetry.Counter
+
+	reinitFailed      *telemetry.Counter
+	processScanFailed *telemetry.Counter
+	callbackExecuted  *telemetry.Counter
+}
+
+func (pmt *processMonitorTelemertry) initialize() {
+	metricGroup := telemetry.NewMetricGroup(
+		"process.monitor",
+		telemetry.OptPayloadTelemetry,
```

Comment:
not all telemetry should be with `OptPayloadTelemetry`
@p-lambert please review and give your suggestions

Reply:
I ask @p-lambert and changed to Prometheus